pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -223,9 +223,9 @@ func generateMetadataListMatcher(keys []string, v string) *metadata.MetadataMatc
 	}
 }
 
-func createStringMatcher(v string, forceRegexPattern, forTCPFilter bool) *metadata.StringMatcher {
+func createStringMatcher(v string, forceRegexPattern, prependSpiffe bool) *metadata.StringMatcher {
```

Comment:
Please also change all the reference to use the new param name in comment, like the one in line 269.
from `/* forTCPFilter */` to `/* prependSpiffe */`

Reply:
Addressed in 5858330.